pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md

```diff
@@ -0,0 +1,112 @@
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+|**Minimum** | 1.21.6 |
+|**Maximum** | 1.21.x |
+|**Default** | 1.21.6 |
+
+### New features and capabilities
+
+#### DKP Major Version Upgrade
+
+Konvoy and Kommander 2.1 represent a major version upgrade that moves forward DKP architecture to give you access to D2iQ's next generation centralized Kubernetes and smart cloud native applications. It incorporates ClusterAPI as a major re-architecture in its management of production Kubernetes clusters. [ClusterAPI](https://cluster-api.sigs.k8s.io/introduction.html), or CAPI, enables declarative creation, configuration, and management of clusters. Declarative mode is a Kubernetes best practice that simplifies lifecycle tasks, making them more consistent and more repeatable. 2.1 enhances your existing clusters to use a new architecture.
+
+#### FIPS Support
+
+In Konvoy 2.1, customers who require FIPS 140-2 encryption ciphers, have an easier way to validate that specific components and services are FIPS-compliant according to the NIST FIPS 140-2 standard by checking the signatures of the files against a signed signature file. For more information on running FIPS validation commands, see Validate FIPS in Cluster.
+
+For more information on using FIPS with Konvoy, see [FIPS 140-2 Compliance](../fips/)
+
+#### Diagnostics Bundle
+
+In DKP 2.1, we provide you a single command for collecting metrics and generating a diagnostic bundle. This allows you to easily upload the bundle to our support team, resulting in less downtime. The diagnostic bundle includes node-level instrumentation around CPU, memory, and disk usage, OS health.
+
+For more information on creating and sending a Konvoy Diagnostics Bundle, see [Generate a Support Bundle](../troubleshooting/generate-a-support-bundle).
+
+#### Support for NVIDIA 470 Driver
+
+Konvoy 2.1 supports the upgraded Nvidia 470.x driver and DCGM exporter 2.2.9, which enables technology for running DKP on NVIDIA DGX. NVIDIA DGX customers now have a robust Kubernetes platform in Konvoy and Kommander that vastly simplifies the process of getting the infrastructure needed for applications up and running on this powerful hardware platform.
+
+For more information on using Nvidia with Konvoy, see the [GPU](../choose-infrastructure/aws/gpu) documentation.
+
+#### Easier Air-gapped deployments
+
+Konvoy 2.1 comes with an easier way to deploy in an air gapped environment. Rather than one large package that includes the requirements for all air gapped environments, you can now create a smaller package that is specific to your requirements and can fit on a laptop or USB.
+
+#### DKP Licensing through Amazon Marketplace
+
+You can purchase a license for Konvoy or Kommander through the AWS Marketplace, and then add it to Kommander. In the Kommander UI, you can see information such as the license status (valid or expired), the license capacity (number of cores or clusters), and expiration date.
+
```

Comment:
```suggestion

### Fixes and Improvements

#### DKP delete fails in AWS

Fixed an issue where the 'dkp delete' command could fail with a SIGSEGV when attempting to delete a DKP cluster from AWS where you have permanent credentials. (COPS-7109)

#### DKP installer fails to follow respective AWS --region or Azure --location flags

Fixed an issue where the AWS `--region` or Azure `--location` installer flags were not being enforced in the target cluster. (COPS-7101)

#### DKP move command not moving some items

We corrected an issue where the `PreprovisionedInventory` object and SSH key secret were not moved to the target cluster when making the cluster self-managing.(COPS-7079)

```

Reply:
agree

Reply:
Not adding these commits, putting them into their appropriate place in 2.1.1 index.md.